.github/workflows/ci.yml

```diff
@@ -55,4 +55,28 @@ jobs:
 
       - name: Run pipeline with additional params (skip library generation)
```

Comment:
You should add the test profiles in the `env` `matrix` so that they can run in parallel and not one after another. Check [this](https://github.com/nf-core/sarek/blob/master/.github/workflows/ci.yml) example.